sshcode.go

```diff
@@ -79,9 +79,17 @@ func sshCode(host, dir string, o options) error {
 
 	flog.Info("starting code-server...")
 
-	if o.localPort == "" {
-		o.localPort, err = randomPort()
+	var bindHost string
+	var bindPort string
+	bindHost, bindPort, err = net.SplitHostPort(o.bindAddr)
```

Comment:
you can short assign this `:=`

Reply:
I can't because of the reassignment of err

Reply:
Try it :)